cli/src/commands/whoami.rs

```diff
@@ -176,7 +176,7 @@ pub async fn run(State(env): State<CliEnv>) {
 
     c_println!();
     // Get admin client, don't fail completely if we can't get one!
-    if let Ok(client) = crate::clients::MetasClient::new(&env) {
+    if let Ok(client) = crate::clients::AdminClient::new(&env).await {
```

Comment:
Do you think we should print the supported version in whoami's output?

Reply:
It probably wouldn't hurt. Will add it.